src/flexmeasures_client/response_handling.py

```diff
@@ -0,0 +1,51 @@
+
+from aiohttp import ContentTypeError
+import asyncio
+
+
+async def check_response(self, response):
```

Comment:
I would say in the docstring that this function is a class method of `FlexMeasuresClient` class.

Reply:
Maybe, It would be nice to add a type hint for `self `: 
```python
from typing import Type
async def check_response(self : Type[FlexMeasuresClient], response)  
```